RSignatureVerifier/Bundle+Environment.swift

```diff
@@ -0,0 +1,40 @@
+extension Bundle: EnvironmentSetupProtocol {
+    var valueNotFound: String {
+        return "NONE"
+    }
+
+    func osVersion() -> String {
+        return UIDevice.current.systemVersion
+    }
+
+    func deviceModel() -> String {
+        var systemInfo = utsname()
+        uname(&systemInfo)
+        return String(bytes: Data(bytes: &systemInfo.machine, count: Int(_SYS_NAMELEN)), encoding: .ascii)!.trimmingCharacters(in: .controlCharacters)
+    }
+
+    func sdkName() -> String {
+        return "Signature Verifier"
+    }
+
+    func sdkVersion() -> String {
```

Comment:
It can be a computed property:
var sdkVersion: String {...}

Reply:
done